### PR TITLE
First pass at testing

### PR DESCRIPTION
**Pull request by levlaz**
- Unit/Integration(ish) tests for Home Controller
- Simple test for the date filter
- Dependencies for Karma and Jasmine
- circle.yml file for CI

Fix https://github.com/standardnotes/web/issues/37

### CODE REVIEW

**Comment by levlaz** [comment]
@mobitar Yay CircleCI is building PRs :)

File: Gemfile [hunk 1]
```diff
@@ -33,6 +33,8 @@ group :development, :test do
 
   gem 'puma'
 
+  gem 'haml'
```
Comment (levlaz) [inline Gemfile]: I was getting the same error locally when I set up the project for the first time. I think this was simply missed somehow?

https://circleci.com/gh/standardnotes/web/3

```
Running "haml:dist" (haml) task
Warning: You need to have Ruby and Haml installed and in your PATH for
this task to work. More info:
https://github.com/jhchen/grunt-haml2html Use --force to continue.
```

**Comment by mobitar** [comment]
This is awesome! Thanks Lev. Will review this first thing tomorrow.

File: Gemfile (hunk 1 above)
Comment (mobitar) [inline Gemfile]: Hmm strange, always worked for me without this. I might have installed it manually on the system though.

**Pull request merged**